cypress/integration/auth.spec.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
```

Comment:
Cypress should include its types, so I haven't figured out why this line is necessary yet.